www/js/lib/uiUtil.js

```diff
@@ -171,6 +171,174 @@ function slideAway (e) {
     }
 }
 
+/*
+  * Returns a list of headings from an article
+    * @param {String} the page for which table of cotents needs to be listed
+    * @returns {List} a list of all headings as objects
+*/
```

Comment:
```suggestion
/**
 * Extracts a list of headings from an article and provides methods to interact with them.
 * @class
 * @param {Document} articleDoc The document object of the article from which headings are to be extracted.
 */
```
Improperly formatted JSDoc.